mappings/net/minecraft/util/ArraySet.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_4706 net/minecraft/util/ArraySet
+	CLASS class_4707 Iterator
+		FIELD field_21566 start I
+		FIELD field_21567 index I
+	FIELD field_21562 sortingFunction Ljava/util/Comparator;
+	FIELD field_21563 entries [Ljava/lang/Object;
+	FIELD field_21564 capacity I
+	METHOD <init> (ILjava/util/Comparator;)V
+		ARG 0 initialCapacity
+		ARG 1 sortingFunction
+	METHOD method_23859 create (I)Lnet/minecraft/class_4706;
+		ARG 0 capacity
+	METHOD method_23862 addIfNotPresent (Ljava/lang/Object;)Ljava/lang/Object;
+	METHOD method_23863 spliceAdd (Ljava/lang/Object;I)V
+		ARG 0 element
+	METHOD method_23865 peek ()Ljava/lang/Object;
```

Comment:
Does this peek the maximum or the minimum value?

Reply:
It uses queue logic. Peek returns `values[0]`.

Reply:
so should be peekMin

Reply:
And what's a `peekMin`? Queues only have one `peek` method and it always returns from the head of the queue.

Reply:
Alternatively: We could call it `head()`, `getHead()`, or `first()`.